PulseRateCheck.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "./monitor.h"
+
+constexpr float MAX_PULSE = 100.0;
+constexpr float MIN_PULSE = 60.0;
+constexpr float PULSE_TOLERANCE = MAX_PULSE * 0.015;
+
+class PulseRateCheck : public VitalCheck {
```

Comment:
Nice abstraction in VitalCheck. Your code structure makes it possible for PulseRateCheck to factor in other parameters (like age) without altering any other classes 👍